src/features/lightspeed/playbookGeneration.ts

```diff
@@ -14,7 +14,6 @@ import { isError, UNKNOWN_ERROR } from "./utils/errors";
 import { getOneClickTrialProvider } from "./utils/oneClickTrial";
 import { LightSpeedAPI } from "./api";
 
-let currentPanel: WebviewPanel | undefined;
 let wizardId: string | undefined;
```

Comment:
`wizardId` also needs to be unique for each instance of the Wizard.

Reply:
[It still is](https://github.com/ansible/vscode-ansible/blob/e04570aa163da35b658512d4366614c0ad49111d/src/features/lightspeed/playbookGeneration.ts#L123).